app/search/dataone.py

```diff
@@ -132,43 +132,40 @@ def convert_result(self, result):
             end = datetime.fromisoformat(result.pop('endDate').rstrip('Z'))
             result['temporal_coverage'] = datetime.date(
                 begin).isoformat() + "/" + datetime.date(end).isoformat()
+
         boundingbox = {'south': result.pop('southBoundCoord', None), 'north': result.pop(
             'northBoundCoord', None), 'west': result.pop('westBoundCoord', None), 'east': result.pop('eastBoundCoord', None)}
 
         if boundingbox:
             # Make a Point if the (north and south) and (east and west) have the same coordinates
             if boundingbox['north'] == boundingbox['south'] and boundingbox['east'] == boundingbox['west']:
-                geometry = Point(coordinates=(boundingbox['north'], boundingbox['east'])
+                bbox = Point(coordinates=(boundingbox['east'], boundingbox['north'])
```

Comment:
PyGeoJSON points are lat, lon, or `east`, `north` for some reason.

By the way, the fact that some things are written `lat`, `lon` and others are written `lon`, `lat` is known as the sideways earth problem in the GIS community, heh.